### PR TITLE
fix(ErrorHandler): custom error handle loop render

### PR DESCRIPTION
**Pull request by Qinyouzeng**

### CODE REVIEW

**Comment by doddgu** [comment]
@Qinyouzeng Azure Static Web Apps CI/CD v1 / Build and Deploy Job (pull_request) Failing after 2m

**Review by capdiem — APPROVED** [review]
LGTM @doddgu

File: src/Doc/Masa.Blazor.Doc/Demos/Components/ErrorHandler/props/onErrorHandleAsync.md [hunk 1]
```diff
@@ -7,8 +7,12 @@ title:
 
 ## zh-CN
 
-自定义异常处理`Func<Exception,Task>`，点击按钮触发异常后，按钮背景色变为红色文字颜色改为白色。
+自定义异常处理`Func<Exception,Task<bool>>`，点击按钮触发异常后，按钮背景色变为红色文字颜色改为白色。
```
Comment (capdiem) [inline src/Doc/Masa.Blazor.Doc/Demos/Components/ErrorHandler/props/onErrorHandleAsync.md:10]: 新文档没改！

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

File: src/Doc/Masa.Blazor.Doc/wwwroot/docs/api/MErrorHandler.zh-CN.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     },
     {
       "Name": "OnErrorHandleAsync",
-      "Type": "Func<Exception,Task>",
+      "Type": "Func<Exception,Task`1>",
```
Comment (doddgu) [inline src/Doc/Masa.Blazor.Doc/wwwroot/docs/api/MErrorHandler.zh-CN.json:14]: `1?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Doc/Masa.Blazor.Doc/wwwroot/docs/api/MErrorHandler.zh-CN.json (hunk 2 above)
Comment (Qinyouzeng) [reply src/Doc/Masa.Blazor.Doc/wwwroot/docs/api/MErrorHandler.zh-CN.json:14]: This is an old document, don't care .

**Pull request merged**